plugins/woocommerce/src/Blocks/BlockTypes/ProductFilterPrice.php

```diff
@@ -212,6 +212,13 @@ function ( $carry, $parsed_block ) use ( $filter_context ) {
 	 * @param WP_Block $block Block instance.
 	 */
 	private function get_filtered_price( $block ) {
+		if ( ! isset( $block->context['filterParams'] ) ) {
+			return array(
+				'min_price' => 0,
+				'max_price' => 0,
```

Comment:
I see in other cases we return empty arrays which I presume is to show "no results". Feels like products may potentially cost `0` and min max would catch them. Nitpicking but what about returning something like to explicitly say "we expect no results"?

```suggestion
				'min_price' => -1,
				'max_price' => -1,
```

Reply:
Yeah, in this case, we can also return an empty array. The intention was to match the shape of the final returned data. But this is a private method, and in the other method calling it, we check for the existence of the key, so an empty array here is better. Thanks for the head up!